test/cppsuite/test_harness/core/configuration.h

```diff
@@ -236,104 +119,28 @@ class configuration {
         return func(value);
     }
 
+    private:
+    static bool config_item_to_bool(const WT_CONFIG_ITEM item);
```

Comment:
Since these are static and don't use any static members, they're basically just free functions. I'd say it makes more sense for them to just go in the `cxx` as an implementation detail and avoid any mention of them in the header.

Reply:
Done.